### PR TITLE
Questions

### PR DESCRIPTION
**Pull request by EricMHernandez**
initial pull request

### CODE REVIEW

**Comment by sheebanair** [comment]
it looks good to me guys. @danitian  @monkeymedic26  I haven't actually commented/approved it, so that you'll can have a look first

File: server/index.js [hunk 1]
```diff
@@ -15,10 +15,58 @@ app.use(morgan('dev'));
...
+
+  axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/qa/questions/${questionId}/answers`, options)
+    .then(({ data }) => {
+      res.status(200).send(data);
+    }).catch((err) => {
+      console.log('there was an error at the server getting answers', err);
+    });
+});
+
+<<<<<<< HEAD
```
Comment (danitian) [inline server/index.js:69]: @EricMHernandez not sure if this is an artifact from when you were resolving merge conflicts?

File: server/index.js [hunk 2]
```diff
@@ -46,6 +94,73 @@ app.get('/products', (req, res) => {
...
+  };
+
+  const { questionId } = req.params;
+
+  axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/qa/questions/${questionId}/report`, req.body, options)
+    .then(() => {
+      res.status(204).end('NO CONTENT');
+    }).catch((err) => {
+      console.log('error reporting the question', err);
+>>>>>>> 59692b0 (implemented helpful button click and report question)
```
Comment (danitian) [inline server/index.js:129]: @EricMHernandez another relic from merge conflict

**Comment by danitian** [comment]
I've commented in a few places but otherwise looks good.

@monkeymedic26 once you finish your review, you can approve.

**Review by monkeymedic26 — APPROVED** [review]
we agreed that it looks good this morning.

**Pull request merged**